packages/vue/src/examples/pages/product/Product.vue

```diff
@@ -616,6 +621,7 @@ export default {
         min-width: calc(
           (375 / 490) * (100vh - 177px)
         ); // (oldWidth / oldHeight) * newHeight = newWidth
```

Comment:
are you sure it's transformed to plain css, not css vars?

Reply:
Haha, thanks I found an error.
Should be:
``` css
min-width: calc((375 / 490) * (100vh - 177px)); // (oldWidth / oldHeight) * newHeight = newWidth
min-width: calc(((var(--vh, 1vh) * 100) - 177px) * (375 / 490));
```